.forge-snapshots/swapExactAmountIn.snap

```diff
@@ -1 +1 @@
-82842
\ No newline at end of file
+105194
```

Comment:
any clue when did this happen??

Reply:
surprisingly, it happened in #110, i don't fully understand how, even the Pool snaps suffered an increase 🤯